libs/sdk-model/src/dashboard/analyticalWidgets.ts

```diff
@@ -48,8 +48,40 @@ export interface IKpiWidgetBase extends IAnalyticalWidget {
      * Drill interactions configured for the kpi widget.
      */
     readonly drills: KpiDrillDefinition[];
+
+    /**
+     * Configuration of the kpi itself
+     */
+    readonly configuration?: IKpiWidgetConfiguration;
+}
+
+/**
+ * @alpha
+ */
+export interface IKpiWidgetConfiguration {
+    description?: IKpiWidgetDescriptionConfiguration;
+}
+
+/**
+ * Configuration of kpi's description
+ * @alpha
+ */
+export interface IKpiWidgetDescriptionConfiguration {
+    /**
+     * whether description should be visible or not
```

Comment:
Please start interface comments with capital letters (ideally with dot at the end, as this is the last convention we should use).